extensions/raycast-chatgpt-api/src/gpt_im.tsx

```diff
@@ -0,0 +1,37 @@
+import { Detail } from "@raycast/api";
+import { useState } from "react";
+import { invokeChatGPT, saveHistory } from "./lib/utils";
+
+/**
+ * 入口文件
+ * @param args
+ * @returns
+ */
+export default function Command(args: any) {
+  // 初始化设置
+  const [isLoading, setIsLoading] = useState(true);
+  const [gptResut, setGptResult] = useState("");
+  const { content } = args.arguments;
+
+  if (isLoading) {
+    const msg = [
+      { role: "system", content: "你将作为我的信息收集助理，解答我提的问题。" },
```

Comment:
I ended up with a Chinese reply to my prompt even though my input was in English (Hello)

Currently, Raycast doesn't support localization and only supports US English. Therefore, please avoid introducing your custom way to localize your extension.